test/node/conversation-moderator.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const assert = require('assert');
+const Circuit = require('../../circuit-node');
+const config = require('./config.json');
+const helper = require('./helper');
+Circuit.logger.setLevel(Circuit.Enums.LogLevel.Error);
+
+let client;
+let user;
+let client2;
+let user2;
+let conversation;
+describe('Moderator Tests', () => {
+    before(async () => {
+        client = new Circuit.Client(config.bot1);
+        user = await client.logon();
+        client2 = new Circuit.Client(config.bot2);
+        user2 = await client2.logon();
+    });
+
+    after(async () => {
+        await client.logout();
+    });
+
+    it('should create a conversation with user2', async () => {
```

Comment:
again we should not duplicate a "create conversation" test. just create the conversation in the before

Reply:
done